src/test/java/com/google/api/generator/engine/writer/JavaWriterVisitorTest.java

```diff
@@ -204,6 +208,86 @@ public void writeVariableExpr_scopedStaticFinalDecl() {
     assertEquals(writerVisitor.write(), "public static final boolean x");
   }
 
+  @Test  
+  public void writeNormalBlockComment() {
+    String content = "this is a test comment";
+    BlockComment blockComment = BlockComment.builder().setComment(content).build();
+    String expected = "/** this is a test comment */\n";
+    blockComment.accept(writerVisitor);
+    assertThat(writerVisitor.write()).isEqualTo(expected);
+  }
+
+  @Test
+  public void writeNormalLineComment() {
+    String content = "this is a test comment";
+    LineComment lineComment = LineComment.builder().setComment(content).build();
+    String expected = "// this is a test comment\n";
+    lineComment.accept(writerVisitor);
+    assertThat(writerVisitor.write()).isEqualTo(expected);
+  }
+
+  @Test
+  public void writeLongLineComment() {
+    String content =
+        "this is a long test comment with so many words, hello world, hello again, hello for 3 times, blah, blah!";
+    LineComment lineComment = LineComment.builder().setComment(content).build();
+    String expected =
+        "// this is a long test comment with so many words, hello world, hello again, hello for 3 times,\n// blah, blah!\n";
+    lineComment.accept(writerVisitor);
+    assertThat(writerVisitor.write()).isEqualTo(expected);
+  }
+
+  @Test
+  public void writeNormalJavaDocComment() {
+    String content = "this is a test comment";
+    String deprecatedText = "Use the {@link ArchivedBookName} class instead.";
+    String paramName = "shelfName";
+    String paramDescription =  "The name of the shelf where books are published to.";
+    String paragraph1 = "This class provides the ability to make remote calls to the backing service through method calls that map to API methods. Sample code to get started:";
+    String paragraph2 = "The surface of this class includes several types of Java methods for each of the API's methods:";
+    String sampleCode = createSampleCode();
+    List<String> orderedlList = Arrays.asList("A flattened method.", " A request object method.", "A callable method.");
+    String throwType = "com.google.api.gax.rpc.ApiException";
+    String throwsDescription = "if the remote call fails.";
+    JavaDocComment javaDocComment =
+        JavaDocComment.builder()
+            .addComment(content)
+            .addParagraph(paragraph1)
+            .addSampleCode(sampleCode)
+            .addParagraph(paragraph2)
+            .addOrderedList(orderedlList)
+            .addSampleCode(sampleCode)
+            .addParam(paramName, paramDescription)
+            .setDeprecated(deprecatedText)
+            .addThrowsText(throwType, throwsDescription)
+            .build();
+    String expected =
```

Comment:
Nit: Please use the `createLines` utility for consistency with the other tests in this file.

Reply:
sounds reasonable! done.